doc/sphinx-guides/source/installation/config.rst

```diff
@@ -208,10 +208,47 @@ As for the "Remote only" authentication mode, it means that:
 
-File Storage: Local Filesystem vs. Swift vs. S3
------------------------------------------------
+File Storage: Using a Local Filesystem and/or Swift and/or S3 object stores
+---------------------------------------------------------------------------
+
+By default, a Dataverse installation stores all data files (files uploaded by end users) on the filesystem at ``/usr/local/glassfish4/glassfish/domains/domain1/files``. This path can vary based on answers you gave to the installer (see the :ref:`dataverse-installer` section of the Installation Guide) or afterward by reconfiguring the ``dataverse.files.directory`` JVM option described below.
+
+Dataverse can alternately store files in a Swift or S3-compatible object store, and can now be configured to support multiple stores at once. With a multi-store configuration, the location for new files can be controlled on a per-dataverse basis.
+
+The following sections describe how to set up various types of stores and how to configure for multiple stores.
+
+Multi-store Basics
++++++++++++++++++
+
+To support multiple stores, Dataverse now requires an id, type, and label for each store (even for a single store configuration). These are configured by defining two required jvm options:
+
+.. code-block:: none
+
+    ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.files.<id>.type=<type>"
+    ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.files.<id>.label=<label>"
+
+For backward compatibility, the id and type should be the same, and the label can be set to the same value as well. For example, the following would define a backward compatible file store:
+
+.. code-block:: none
+
+    ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.files.file.type=file"
+    ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.files.file.label=file"
+  
+or a file or swift store, this is all that is needed for backward compatibility. For an s3 store, any additional options must be changed to conform to the new, more consistent naming convention using a . after the store's id ('s3' for backward compatibility) For example:
```

Comment:
We typically put information like this (what to change after you deploy a new version) in release notes. I guess it's ok to have it here too but I prefer to think of the audience as being someone who is installing Dataverse for the first time. This could be their first day with Dataverse. They probably don't need information about how things used to be (Don't Make Me Think).

Reply:
I'm fine with it being in release notes instead. I figured I'd put it here since I don't know what version this might become part of.